docs/changes.rst

```diff
@@ -395,6 +395,17 @@ This version contains all fixes up to pywbem 0.12.4.
 * Improved the quality of any `ParseError` exception messages when the SAX
   parser detects errors in CIM-XML responses. See issue #1438.
 
+* Added a `ServerWarning` class and its base class `Warning`. The new
+  `ServerWarning` is raised in cases when the WBEM server exhibits some
+  incorrect behavior that is tolerated by pywbem.
+
+* Added support for tolerating ill-formed XML in the CIM-XML response returned
+  by the server from the attempt to invoke the CIM method GetCentralInstances()
+  inside of `WBEMServer.get_central_instances()`. One server was found to
+  return such ill-formed XML. This now causes pywbem to issue a `ServerWarning`
+  and to continue with the next approach for determining the central instances.
+  See issue #1438.
+
```

Comment:
An entry in the change log is missing

Reply:
log entry added

Reply:
Note that I added it under enhancements rather that bugs